test/freenet/client/filter/CSSParserTest.java

```diff
@@ -918,112 +905,107 @@ public void setUp() throws InvalidThresholdException {
 
 	@Test
 	public void testCSS1Selector() throws IOException, URISyntaxException {
+		testCssSelectorFiltering(CSS1_SELECTOR);
+		assertEquals(
+			"key=\"" + CSS_DELETE_INVALID_SELECTOR + "\" value=\"" + filter(CSS_DELETE_INVALID_SELECTOR) + "\" should be \"" + CSS_DELETE_INVALID_SELECTORC + "\"",
+			CSS_DELETE_INVALID_SELECTORC,
+			filter(CSS_DELETE_INVALID_SELECTOR)
+		);
+		assertEquals(
+			"key=\"" + CSS_INVALID_MEDIA_CASCADE + "\" value=\"" + filter(CSS_INVALID_MEDIA_CASCADE) + "\"",
+			"",
+			filter(CSS_INVALID_MEDIA_CASCADE)
+		);
+	}
 
-
-		Collection<String> c = CSS1_SELECTOR.keySet();
-		Iterator<String> itr = c.iterator();
-		while(itr.hasNext())
-		{
-
-			String key=itr.next();
-			String value=CSS1_SELECTOR.get(key);
-			assertTrue("key=\""+key+"\" value=\""+filter(key)+"\" should be \""+value+"\"", filter(key).contains(value));
+	private void testCssSelectorFiltering(Map<String, String> cssSelectorMap) throws IOException, URISyntaxException {
+		for (Entry<String, String> entry : cssSelectorMap.entrySet()) {
+			String key = entry.getKey();
+			String value = entry.getValue();
+			assertTrue("key=\"" + key + "\" value=\"" + filter(key) + "\" should be \"" + value + "\"", filter(key).contains(value));
 		}
+	}
 
-		assertTrue("key=\""+CSS_DELETE_INVALID_SELECTOR+"\" value=\""+filter(CSS_DELETE_INVALID_SELECTOR)+"\" should be \""+CSS_DELETE_INVALID_SELECTORC+"\"", CSS_DELETE_INVALID_SELECTORC.equals(filter(CSS_DELETE_INVALID_SELECTOR)));
-		assertTrue("key=\""+CSS_INVALID_MEDIA_CASCADE+"\" value=\""+filter(CSS_INVALID_MEDIA_CASCADE)+"\"", "".equals(filter(CSS_INVALID_MEDIA_CASCADE)));
+	private void testBadSelectorFiltering(Set<String> badSelectorSet) throws IOException, URISyntaxException {
+		for(String key : badSelectorSet) {
```

Comment:
Inconsistent whitespace after `for`.

Reply:
Fixed, 
manually typed vs auto-formatted ;)